bin/compile

```diff
@@ -87,20 +92,18 @@ def process_backends_as_mvn_profiles(selected_backends):
     for i, backend in enumerate(selected_backends_list):
         selected_backends_list[i] = f"{backend}-backend"
     backend_profiles = ",".join(selected_backends_list)
+
     return backend_profiles
 
+
 def clone_opencl_headers():
     """
     Clone the Khronos OpenCL headers into the OpenCL JNI lib
     """
     current = os.getcwd()
-    directoryName = "tornado-drivers/opencl-jni/src/main/cpp/headers"
-    tmpDirectory = "tempOpenCLHeaderDirectory"
+    directoryName = os.path.join("tornado-drivers", "opencl-jni", "src", "main", "cpp", "headers")
     if not os.path.exists(directoryName):
         ## clone the repo with the OpenCL Headers
-        import shutil
```

Comment:
Why there is no need to create a temporal directory?

Reply:
There is no naming conflict between the top directory of the OpenCL header repository and other pre-existing files. Therefore, I don't think a temporary directory is necessary.

Furthermore, I skipped the removal of the OpenCL header directory after installation, just in case someone wants to look inside later for whatever reason... But after all, it's a matter of taste. If you prefer a temporal directory with removal, I'm fine with that.

Reply:
Ok, thanks for the clarification. Let's keep it then.